wdl-analysis/src/types/v1.rs

```diff
@@ -1432,14 +1443,47 @@ impl<'a, C: EvaluationContext> ExprTypeEvaluator<'a, C> {
                 // Evaluate the argument expressions
                 let mut count = 0;
                 let mut arguments = [const { Type::Union }; MAX_PARAMETERS];
+                let mut arguments_node = Vec::with_capacity(expr.arguments().count());
```

Comment:
I don't think we need this `Vec`.

We can replace it with a call to `if let Some(argument) = expr.arguments().nth(1)`.